coalib/settings/ConfigurationGathering.py

```diff
@@ -252,7 +257,8 @@ def load_configuration(arg_list,
         user_sections = load_config_file(
             Constants.user_coafile, silent=True)
 
-        default_config = str(base_sections['default'].get('config', '.coafile'))
+        default_config = str(
+            base_sections['default'].get('config', '.coafile'))
```

Comment:
Why have you changed this?

Reply:
When I ran coala it was showing this as error. So I corrected it.

Reply:
i dont think this change is necessary.